<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_email</class>
 <widget class="QWidget" name="Qmain_email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,15">
     <item>
      <widget class="QGroupBox" name="Qbox_actions">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="Qbutton_send">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Trimite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_save">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Salveaza</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_abort">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Renunta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_attach">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Ataseaza</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Qlabel_attachments">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Atasamente:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="Qscroll_attachments">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="Qcontent_attachments">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>92</width>
             <height>466</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="Qlabel_title">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>E-mail service brought to you by The JoestarsÂ®</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="Qbox_stil">
             <property name="title">
              <string>Stil text</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="Qcheck_bold">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Bold</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_italic">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>true</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Italic</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_underline">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Undeline</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_strikeout">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                  <strikeout>true</strikeout>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Strikeout</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="Qbox_font">
             <property name="title">
              <string>Font text</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="Qcheck_calibri">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <strikeout>false</strikeout>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Calibri</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_tnr">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>T. N. R.</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_arial">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Arial</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Qcheck_courier">
                <property name="font">
                 <font>
                  <family>Courier</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Courier</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="Qbox_size_2">
             <property name="title">
              <string>Marime text</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="Qlabel_size">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Marime:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Qsbox_size"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>60</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
               <item>
                <widget class="QLabel" name="Qlabel_to">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Catre:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Qlabel_subject">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Subiect:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QPlainTextEdit" name="Qtext_to">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="cursor" stdset="0">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="Qtext_subject">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="cursor" stdset="0">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="Qtext_message"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="Qline1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
